src/components/App/App.js

```diff
@@ -0,0 +1,61 @@
+import AllMovies from '../AllMovies/AllMovies';
+import MovieDetails from '../MovieDetails/MovieDetails';
+import Navbar from '../Navbar/Navbar';
+import './App.css';
+import { useState, useEffect } from 'react';
+import "@fontsource/monoton";
+import { getData } from '../.././apiCalls/apiCalls'
+import { checkServerError } from '../../helperFunctions';
+import { Route, Routes } from 'react-router-dom'
+
+const App = () => {
+  const [searchValue, setSearchValue] = useState('');
+  const [movies, setMovies] = useState([]);
+  const [allMovies, setAllMovies] = useState([])
+  const [serverError, setServerError] = useState(false);
+  const [waitingForFetch, setWaitingForFetch] = useState(true);
+
+  let chosenMovieId
```

Comment:
This doesnt look needed.

Reply:
Ugh. Yes. Missed it in the refactor. Thanks for the catch!